net-mail/mu/mu-1.4.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools elisp-common
+
+DESCRIPTION="Set of tools to deal with Maildirs, in particular, searching and indexing"
+HOMEPAGE="http://www.djcbsoftware.nl/code/mu/ https://github.com/djcb/mu"
```

Comment:
https://djcbsoftware.nl/code/mu/

Reply:
Thanks. Changed the homepage URL and manual URL in info message in all mu ebuilds.